hycom/MSCPROGS/src/Hyc2proj/MODEL.CPP

```diff
@@ -1 +1 @@
-#undef TP4
+#undef TP4    /* This was the old HYCOM CPP key */
```

Comment:
I would instead of "old HYCOM" use "HYCOMv2.2.37/HYCOM2.2.12" as Laurent suggested: so I sugeest:
"Define this CPP-key to process files from HYCOMv2.2.37/HYCOM2.2.12"